create_board.py

```diff
@@ -0,0 +1,53 @@
+from random import randrange, choice
+from string import ascii_lowercase
+
+
+"""
+creates board file
+"""
+
+def create_board(file_path, board_size=100, number_of_snakes=10, number_of_ladders=10, number_of_players=3):
```

Comment:
This function is not being used anywhere?

Reply:
this was the part of optional task. It is used in main.py but is commented.

Reply:
Even if its for the optional task it should be part of its relevant class.

Reply:
It generates the board input file. It is a function that will be called before every function/class. I don't think so there is any proper class for this in my code. Is it a good practice just to create a a new class for just one function?

Reply:
If you're using OOP, then everything should be in a class.